crates/sargon/src/wrapped_radix_engine_toolkit/low_level/v1/transaction_manifest/execution_summary/transaction_manifest_execution_summary.rs

```diff
@@ -216,7 +216,7 @@ mod tests {
                 [acc_g2], // addresses_of_accounts_requiring_auth
                 [],               // addresses_of_identities_requiring_auth
                 [],               // newly_created_non_fungibles
-                [ReservedInstruction::AccountLockFee, ReservedInstruction::AccountUpdateSettings],
+                [ReservedInstruction::AccountLockFee],
```

Comment:
why has this changed?

Reply:
AccountUpdateSettings is no longer a reserved instruction